std/experimental/checkedint.d

```diff
@@ -368,6 +370,14 @@ if (isIntegral!T || is(T == Checked!(U, H), U, H))
         assert(a == 4242);
     }
 
+    ///
+    unittest
+    {
+        Checked!long a, b;
+        a = b = 3;
+        assert(a == 3 && b == 3);
```

Comment:
The return type now is `this`, does this fix also work for mixed native & checked chain assignment? i.e. add to unittest:

```
  long la, lb;
  Checked!long ca, cb;

  la = ca = lb = cb;  // mixed chain assign
  ca = la = cb = lb;
```

Reply:
please mind this